google-cloud-graalvm-samples/graalvm-samples-spring/pom.xml

```diff
@@ -16,7 +16,7 @@
   <artifactId>graalvm-samples-spring</artifactId>
 
   <properties>
-    <spring-boot.version>2.4.0-M3</spring-boot.version>
+    <spring-boot.version>2.5.0</spring-boot.version>
     <spring-cloud-gcp.version>1.2.8.RELEASE</spring-cloud-gcp.version>
```

Comment:
Is this the correct version of `spring-cloud-gcp`?  Using start.spring.io it gives a project with 2.0.0, and I was able to use Spring Boot 2.4.5 with Spring Native 0.9.2 https://github.com/GoogleCloudPlatform/spring-cloud-gcp/issues/379#issuecomment-846650626